### DISCUSSION TITLE
Unable to print certain FontAwesome icons

### DISCUSSION THREAD

**Discussion by Mecanik** [razonyang/hugo-theme-bootstrap#543] [Q&A] [answered]
Hi,

I noticed you are using @fortawesome library which I'm quite familiar with, but I`m unable to understand why most icons do not convert to SVG's.

For example, I cannot print the following in the menu:

```
<i class="fas fa-address-card"></i>
<i class="fas fa-newspaper"></i>
<i class="fas fa-project-diagram"></i>
<i class="fas fa-tools"></i>
```

Could someone please elaborate on this?

Many thanks

**Comment by razonyang** [comment, marked as answer]
You'll need to import the icons you need first(`assets/icons/custom.js`).

See also https://hbs.razonyang.com/en/docs/look-and-feel/#icons.

**Reply by Mecanik** [reply]
Thank you, I did not notice this section. Works like a charm.
